scripts/migration-project_15_0_0/perf/get_submissions_info.py

```diff
@@ -0,0 +1,54 @@
+"""
+This script was ran locally with modified version of ingest-core containing an endpoint to get an array assay process ids.
+"""
+import logging
+
+import requests
+
+from migration.submission import Submission
+from migration.util import load_json, write_json
+
+FILENAME = 'submission-envelopes.json'
+
+
+class Submission2(Submission):
```

Comment:
What does `Submission2` mean?

Reply:
This is @aaclan-ebi script. I branched off her branch

Reply:
I honestly can't remember, I think it fetches more data (assays) than Submission. I also modified the core locally so this script won't work against the deployed one. Just added it for reference.